### ISSUE TITLE
Calc name

### ISSUE THREAD

**Issue by 1qschool1** [Timendus/ticalc-usb#68]
When a calculator is renamed with a cermaster ticalc link doesnt identify it as a ti 84 plus ce just some new calculator.

**Comment by adriweb** [comment]
Yeah kinda similar to #60 , the fix would be not to rely on the cert name.
